### PR TITLE
En_Nwc

### PR DESCRIPTION
**Pull request by petrie911**
Incomplete actor. I wonder if they went in child Lon Lon, where the flock of cuccos is as an adult.

### CODE REVIEW

File: src/overlays/actors/ovl_En_Nwc/z_en_nwc.c [hunk 1]
```diff
@@ -21,25 +51,200 @@ const ActorInit En_Nwc_InitVars = {
...
+    chick->floorY = func_8003C940(&globalCtx->colCtx, &outPoly, &bgId, &outPos);
+    dy = chick->floorY - chick->pos.y;
+    if ((0.0f <= dy) && (dy < 40.0f)) {
+        chick->pos.y = chick->floorY;
+        chick->bgFlags |= CHICK_BG_FLOOR;
+    }
+}
+
+void EnNwc_ChickFall(EnNwcChick *chick, EnNwc *this, GlobalContext *globalCtx) {
+    chick->velY -= 0.1f;
```
Comment (fig02) [inline src/overlays/actors/ovl_En_Nwc/z_en_nwc.c:97]: formatter

File: src/overlays/actors/ovl_En_Nwc/z_en_nwc.c [hunk 2]
```diff
@@ -21,25 +51,200 @@ const ActorInit En_Nwc_InitVars = {
...
+
+    gSPDisplayList(dList1++, D_06000840);
+    gSPDisplayList(dList2++, D_06000BF0);
+    gSPDisplayList(dList3++, D_06000C88);
+
+    chick = this->chicks;
+    for(i = 0; i < this->count; i++, chick++) {
+        if(chick->type != CHICK_NONE) {
+            Mtx* tempMat;
+
```
Comment (fig02) [inline src/overlays/actors/ovl_En_Nwc/z_en_nwc.c:166]: i call this mtx personally

File: src/overlays/actors/ovl_En_Nwc/z_en_nwc.c (hunk 1 above)
Comment (petrie911) [reply src/overlays/actors/ovl_En_Nwc/z_en_nwc.c]: Formatted. Also formatted the rest of the repo accidentally.

**Review by NEstelami — APPROVED** [review]

**Review by fig02 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Nwc/z_en_nwc.h [hunk 3]
```diff
@@ -5,10 +5,32 @@
 #include "global.h"
 
 struct EnNwc;
+struct EnNwcChick;
+
+typedef void (*EnNwcUpdateFunc)(struct EnNwc*, GlobalContext*);
+typedef void (*EnNwcChickFunc)(struct EnNwcChick*, struct EnNwc*, GlobalContext*);
+
+typedef struct EnNwcChick{
```
Comment (Roman971) [inline src/overlays/actors/ovl_En_Nwc/z_en_nwc.h:13]:
```suggestion
typedef struct EnNwcChick {
```

**Review by Roman971 — APPROVED** [review]

**Pull request merged**